cs-vmware/src/main/java/io/cloudslang/content/vmware/entities/Uploader.java

```diff
@@ -0,0 +1,43 @@
+package io.cloudslang.content.vmware.entities;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLSession;
+import java.io.IOException;
+import java.net.URL;
+
+public class Uploader {
+
+    public static HttpsURLConnection getHTTPSUploadConnection(final URL url, final int chunkLength,
+                                                              final long contentLength, final boolean put) throws IOException {
+        final HttpsURLConnection connection = getBasicHTTPSConnection(url);
```

Comment:
We might in case of exception leak connections if we don't have try finally block